skeleton_scary_game.py

```diff
@@ -0,0 +1,151 @@
+                else:
+                    print("You don't have any batteries \n")
+                    pass
+            elif need_help.lower() == "n":
+                pass
+
+            # Display the available paths
+            for i, path in enumerate(self.paths):
+                print(f"{i + 1}. {path}\n")
+            
+            # User picks their path
+            loop_count += 1
+            user_choice = int(input(f"Tunnel #{loop_count} "
+                                            f"Walk your path => "))
+
+
+            # If the user picks a path number that is not valid, they have to re-pick
+            if user_choice < 1 or user_choice > 3:
+                print("Invalid choice! Please enter a valid path number.")
+                continue
+
+            # Evaluate the chosen path
+            chosen_path = self.paths[user_choice - 1]
+
+            # If the user picks this path, then it will call the demon class
+            if chosen_path == demon_path:
+                self.demon.death()
+            
+            # If the user picks this path it will call the item class.
+            elif chosen_path == item_path:
+                self.item.collect_item()
+            
+            # If the user picks this path then nothing happens but just run the loop again.
+            elif chosen_path == lore_path:
+                self.lore.lore_play()
+            
+        # Once the user has survived 7 rounds then they win the game and get their daugther back.
+        if loop_count >= 7:
+            print("The demon said thank you for playing and returned your child")
+            exit(0)
+
+# The father class, and the only thing this class can do is use and item.
+class father(object):
+    
+    # This is where the item is acutally used and then removed after the usage.
+    def use_item(self,bagpack,demon):
+        print(f"The demon is on {demon}. \n")
+        bagpack.remove("Battery")
+
+
+# This is the evil demon object, and it will be sent to the game_engine 
+class evil_demon(object):
+    
+    #If  the user picks this path, the user dies 
+    def death(self):
+        print("The demon ate you for dinner and let your child go back home parentless.")
+        exit(1)
+    
+
+# This is the item object, and will be sent to the game engine as well.
+class item(object):
+
+    # These are the variables that are in this class. and will only be in this class.
+    items_list = ["Battery"]
+    users_bagpack = ["Battery"]
+
+    # If the user picks this path, the user collects the item and will be sent to his bagpack
+    def collect_item(self):
+        prize_item = random.choice(self.items_list)
+        print(f"You have survived and you have found => {prize_item}")
+        self.users_bagpack.append(prize_item)
+        print(f"You have {self.users_bagpack} \n")
+
+        return self.users_bagpack
+
+# This is the lore object, and will be sent to the game engine also.
+class lore(object):
```

Comment:
This class name must be capitalized and the description added as a doc string as mentioned above